docs/01.02.layer.md

```diff
@@ -49,73 +49,73 @@ You can create a new image or open an image as a layer.
 
 #### Image Atlas
 
-When your create new image using Ucupaint, it will give you option to use image atlas.
+When your create new image using Ucupaint, it will give you the option to use an image atlas.
 
 |![03.layer.06](source/03.layer.06.png)|
 |:--:|
 |Image atlas checkbox on new image layer options| {align=center}
 
-If you turned on Image Atlas, newly created layer will use portion of larger image. Since blender material has limited number of images that can be used in a material (8-16 depending on your system), image atlas will use special image that can be used as several layer or masks at the same time. The system works automatically, so you don't have to worry about it.
+If you turn on Image Atlas, the newly created layer will use portion of a larger image. Since Blender materials have limited number of images that can be used in a material (8-16 depending on your system), image atlas will use a special image that can be used as several layers or masks at the same time. This system works automatically, so you don't have to worry about it.
 
 ### Vertex Color
 
-Vertex color can be used for a layer. There's also special vertex color editor on edit mode
+Vertex color can be used as a layer. There's also special vertex color editor in edit mode.
 
 |![type:video](source/03.layer.07.mp4)|
 |:--:|
 |Vertex color editor on edit mode| {align=center}
 
 ### Solid Color
 
-Solid Color layer contains only single color, can be useful if you want to paint the mask first then edit the solid color later
+Solid Color layer contains only single color, can be useful if you want to paint the mask first then edit the solid color later.
```

Comment:
One thing I wasn't sure about is that I made it a rule to put periods after every regular sentence, which was missing a lot of the times. But maybe you have different rules on where to put punctuation. If so let me know. 🙂